processor/src/operations/comb_ops.rs

```diff
@@ -202,18 +203,25 @@ mod tests {
         // --- setup memory -----------------------------------------------------------------------
         let ctx = ContextId::root();
         let tztgz = rand_array::<Felt, 4>();
-        process.chiplets.write_mem(
-            ctx,
-            inputs[2].as_int().try_into().expect("Shouldn't fail by construction"),
-            tztgz,
-        );
+        process
+            .chiplets
+            .write_mem(
+                ctx,
+                inputs[2].as_int().try_into().expect("Shouldn't fail by construction"),
+                tztgz,
+            )
+            .unwrap();
 
         let a = rand_array::<Felt, 4>();
-        process.chiplets.write_mem(
-            ctx,
-            inputs[1].as_int().try_into().expect("Shouldn't fail by construction"),
-            a,
-        );
+        process
+            .chiplets
+            .write_mem(
+                ctx,
+                inputs[1].as_int().try_into().expect("Shouldn't fail by construction"),
+                a,
+            )
+            .unwrap();
+        process.execute_op(Operation::Noop).unwrap();
```

Comment:
Q: Is this related to your latest comment?

Reply:
Yup!